docs/octopus-rest-api/index.md

```diff
@@ -11,12 +11,18 @@ The Octopus REST API is designed:
 1. To be friendly and easy to figure out.
 2. To be [hypermedia driven](http://en.wikipedia.org/wiki/HATEOAS), using links and the occasional [URI template](http://tools.ietf.org/html/rfc6570) ([read more](https://github.com/OctopusDeploy/OctopusDeploy-Api/wiki/Links)).
 3. To be comprehensive - 100% of the actions that you perform via the Octopus UI can be performed via the API.
-4. To have a nice [client library](http://www.nudoq.org/#!/Projects/Octopus.Client) for .NET [available via NuGet](http://www.nuget.org/packages/Octopus.Client/).
+4. To provide a great developer experience through [API clients](#api-clients) and [detailed examples](/docs/octopus-rest-api/examples/index.md).
 
-:::success
-**Using C#?**
-If you plan to use the REST API from C# or another .NET project, see the [Octopus.Client](/docs/octopus-rest-api/octopus.client/index.md) assembly.
-:::
+## API Clients
+
+Octopus provides API clients for popular programming languages and runtime environments. The source code for these clients is hosted on GitHub:
+
+- [Go API Client for Octopus Deploy](https://github.com/OctopusDeploy/go-octopusdeploy)
+- [Java API Client for Octopus Deploy](https://github.com/OctopusDeployLabs/java-octopus-deploy)
+- [OctopusClients](https://github.com/OctopusDeploy/OctopusClients) (C#-based API client for .NET)
```

Comment:
I know the client has historically been called the `Octopus Client` but I wonder if we should break from the past here and just follow the other language conventions, e.g.:

```suggestion
- [.NET C# API Client for Octopus Deploy](https://github.com/OctopusDeploy/OctopusClients)
```